### ISSUE TITLE
Bug: "Boards" category is sorted in alphabetical order by default

### ISSUE THREAD

**Issue by wuwinson** [mattermost/focalboard#4138]
## Steps to reproduce the behavior

1. Go to the sidebar and add new categories, some starting with the letter A and some with the letter C
2. See error - the default "Boards" category is sorted in between the new custom categories

## Expected behavior

The "Boards" category should always appear on the bottom, unless the user drags and drops the category after DND is supported.

## Screenshots (optional)

![Screen Shot 2022-11-03 at 2 42 13 PM](https://user-images.githubusercontent.com/93531870/199807481-fc3fec2d-2744-46bd-8bac-148dbe08bfad.png)

## Edition and Platform

 - Edition: Mattermost Boards (plugin)
 - Version: v7.5 Bug Bash
 - Browser and OS: Chrome on Mac

**Comment by wuwinson** [comment]
@harshilsharma63 Will this be fixed after you merge in your code?

**Comment by harshilsharma63** [comment]
@wuwinson yes, exactly. Once I get the whole LHS DND PR merged in, it will be fixed and work as per the spec.
